SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialWorker.cpp

```
@@ -0,0 +1,45 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialWorker.h"
+
+#include "SpatialConstants.h"
+
+#include "Engine/World.h"
+
+
+bool USpatialWorker::CanHaveAuthority(const AActor* Actor)
+{
+	if (Actor == nullptr)
+	{
+		return false;
+	}
```

why should this happen? should return an error?